cypress/e2e/shared/arduino.test.ts

```diff
@@ -0,0 +1,83 @@
+describe('Arduino onboarding', () => {
+  beforeEach(() => {
+    cy.flush()
+    cy.signin()
+    cy.setFeatureFlags({firstMile: true, onboardArduino: true})
```

Comment:
setting `firstMile: true` is no longer needed. the `firstMile` flag has been removed